<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChemistryLab-Nhom07</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/lab-color.png</normaloff>icon/lab-color.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLab">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Laboratory</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/lab.png</normaloff>icon/lab.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>225</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpProduceSteps">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Các bước thực hiện</string>
    </property>
    <widget class="QPlainTextEdit" name="txtProduceSteps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>311</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpEquations">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Phương trình hóa học</string>
    </property>
    <widget class="QToolButton" name="btnInputEquationsFromFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="Open a text file contains the equations"/>
     </property>
     <property name="text">
      <string>Chọn từ file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputEquations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpProblem">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Điều chế</string>
    </property>
    <widget class="QLineEdit" name="inputSourceChemistry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputTargetChemistry">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/right-arrows.png</normaloff>icon/right-arrows.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>16</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtTitle">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ĐIỀU CHẾ CHẤT HÓA HỌC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtProcessing">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Processing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>grpProblem</zorder>
   <zorder>grpEquations</zorder>
   <zorder>grpProduceSteps</zorder>
   <zorder>btnLab</zorder>
   <zorder>txtTitle</zorder>
   <zorder>txtProcessing</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAuthors">
    <property name="title">
     <string>Nhóm 07</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRefresh">
    <property name="title">
     <string>Thí nghiệm mới</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuChemistryLab">
    <property name="title">
     <string>ChemistryLab</string>
    </property>
   </widget>
   <addaction name="menuRefresh"/>
   <addaction name="menuAuthors"/>
   <addaction name="menuChemistryLab"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
